.github/workflows/auto-delete-on-schedule.yml

```diff
@@ -56,26 +77,6 @@ jobs:
   delete-check:
     needs: [rosa-delete, aurora-delete, route53-delete]
     runs-on: ubuntu-latest
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
-
-      - name: Configure AWS CLI
-        run: |
-          aws configure set aws_access_key_id ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws configure set aws_secret_access_key ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws configure set region ${{ vars.AWS_DEFAULT_REGION }}
-
-      - shell: bash
-        run: ./provision/aws/aws_check_vpcs.sh
-
-      - shell: bash
-        if: ${{ always() }}
-        run: ./provision/aws/aws_check_subnet_groups.sh
```

Comment:
aws_check scripts are not invoked anymore.

Reply:
It seems moving up and down of yaml blocks does not work very well in Intellij.